src/one_dragon/base/operation/one_dragon_context.py

```
@@ -297,11 +297,55 @@ def init(self) -> None:
+            cls_file = inspect.getfile(self.__class__)
+            src_dir = file_utils.find_src_dir(cls_file)
+            if src_dir is None:
+                return
+            project_root = src_dir.parent
+            git_dir = project_root / ".git"
+            githooks_dir = project_root / ".githooks"
+            
+            if git_dir.exists() and githooks_dir.exists():
+                # 检查当前的 hooksPath
+                result = subprocess.run(
+                    ["git", "config", "core.hooksPath"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)